pax-utils.yaml

```diff
@@ -30,7 +26,7 @@ environment:
 pipeline:
   - uses: fetch
     with:
-      uri: https://dev.gentoo.org/~xen0n/distfiles/pax-utils-${{package.version}}.tar.xz
+      uri: https://gitweb.gentoo.org/proj/pax-utils.git/snapshot/pax-utils-${{package.version}}.tar.gz
```

Comment:
`1.3.5` is now also available but not under the original uri above.  Moving to this uri means we can automate future version updates.